### DISCUSSION TITLE
Multiple annotators per scan

### DISCUSSION THREAD

**Discussion by malteekj** [MIC-DKFZ/nnUNet#1229] [Q&A]
Hi, is it possible to train with multiple annotators for each scan? In my case I have 3 annotators per scan and I would like to train where the annotator mask is randomly drawn during training, so that the network learn from all annotators.

Is this possible, besides just copying the image data 3 times and pairing the copies up with each annotator?

**Comment by JackRio** [comment]
You can always implement your own dataloader and implement it so it picks random annotator on the fly
